res/main.qml

```diff
@@ -0,0 +1,36 @@
+import QtQuick 2.12
+import QtQuick.Controls 2.12
+import Mixxx 1.0
+
+ApplicationWindow {
+    id: window
+    width: 34
+    height: 100
+    visible: true
+
+    Slider {
+        id: slider
+        Control {
+            id: volume
+            group: "[Channel1]"
+            key: "volume"
+            parameter: slider.value
+        }
+        value: volume.parameter
```

Comment:
This can be removed as `parameter: slider.value` is a two way binding

Reply:
That breaks controller input

Reply:
You're right. Sorry I was rushing and had some debug statements in which confused me.